src/plugins/intel_gpu/src/graph/graph_optimizer/compile_graph.cpp

```diff
@@ -58,6 +58,11 @@ void compile_graph::run(program& p) {
             can_select_impl = false;
         }
 
+        // TODO: Remove this WA once we have shape agnostic conv kernl with specified auto_pad attributes
+        if (node->is_type<convolution>() && node->is_dynamic() && !node->as<convolution>().use_explicit_padding()) {
+            can_select_impl = false;
+        }
```

Comment:
for deconv, binary_convolution, too, right?

Reply:
Currently, only convolution specifies auto_pad attribute from ngraph op. deconvolution or binary_convolution is always explicit padding.